convert_matrices.py

```diff
@@ -612,13 +609,12 @@ class GSE113197(Converter):
 
     def _convert(self):
         # row compatibility verified using ~/load-project.nadove/check_genes
-        # noinspection PyUnreachableCode
         self._convert_matrices(
             CSVPerCell(
                 'GSE113197_RAW',
                 path_filter=lambda p: not p.name.endswith('Matrix.txt.gz'),
                 sep=' ',
-                entry_filter=methodcaller('__delitem__', 0)
+                entry_filter=methodcaller('__delitem__', 1)
```

Comment:
I think this needs a comment explaining the intent. I don't understand what it does and I don't understand why the index changed.

Reply:
And consider a lambda that calls `pop`. 

`methodcaller('__delitem__', 0)`
`lambda entries: entries.pop(0)`

is the same length (and I'm being extra verbose with the argument name) doesn't call through to the dunder method which you shouldn't call directly.

Reply:
We cannot use `pop` because it returns the popped value. CSV conversion uses the return value of the filter function to decide whether to drop the row, and values other than `True`, `False`, or `None` result in an error (https://github.com/DailyDreaming/load-project/blob/5c184fa321edb1c49bd685cb8b774788ed74566c/csv2mtx.py#L70)

I will explain the 0/1 indexing